permissions/plugin-zoom.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "zoom"
+github: "jenkinsci/zoom-plugin"
+paths:
+- "org/jenkins-ci/plugins/zoom"
+developers:
+- "zoomdevplatform"
```

Comment:
https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=zoomdevplatform is 404

Reply:
@batmat Fixed. Thanks!

Reply:
Hi @batmat, can you help to try again?